spec/buffered-process-spec.coffee

```diff
@@ -117,6 +111,7 @@ describe "BufferedProcess", ->
       expect(stdout).toEqual ''
 
   it "calls the specified stdout callback only with whole lines", ->
+    # WINSPEC: Fails because command is too long, /bin/echo is *nix only, odd newlining and quoting
```

Comment:
Can we use any other command that works on Windows too and that verifies the same behavior? Or, alternatively, we can also write a different spec that runs on Windows only: 

```coffee
describe "on #darwin and #linux", ->
  it "calls the specified stdout callback only with whole lines", ->

describe "on #windows", ->
  it "calls the specified stdout callback only with whole lines", ->
```